### ISSUE TITLE
AOSP theme

### ISSUE THREAD

**Issue by menny** [AnySoftKeyboard/LanguagePack#260]
see https://github.com/AnySoftKeyboard/AnySoftKeyboard/issues/1557

**Comment by menny** [comment]
https://github.com/LineageOS/android_packages_inputmethods_LatinIME/tree/lineage-16.0/java/res

**Comment by returntrip** [comment]
Thanks for implementing this theme! To complete it we also need a new bottom row that imitates the Gboard style.

**Comment by menny** [comment]
makes sense. Can you provide me with the requirements of that bottom row?

**Comment by returntrip** [comment]
This is how it looks for me 

[![Screenshot-20190502-203733-01.jpg](https://i.postimg.cc/0Nmdytm5/Screenshot-20190502-203733-01.jpg)](https://postimg.cc/SXSzZfDH)

**Comment by tnyeanderson** [comment]
Is there a PR for this yet? Maybe I could help out if needed. Would love to get this theme working!
